package.json

```diff
@@ -34,16 +34,18 @@
 		"node": "^18.12"
 	},
 	"dependencies": {
-		"@companion-module/base": "~1.0",
-		"rxjs": "~7.5.0",
-		"soundcraft-ui-connection": "1.4.0"
+		"@companion-module/base": "^1.4.1",
+		"bufferutil": "^4.0.7",
```

Comment:
do you need/use bufferutil and utf-8-validate?
From memory they are both native modules and dont get packaged up properly by our build process

Reply:
I changed the build process for the connection lib. It works in my own Node experiments. However, the conpanion module build complained about those two so I added them here.

Reply:
ah right. they are optional dependencies of `ws`, so webpack will warn about them. 
either way is fine really